torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7210,6 +7210,37 @@ def sample_inputs_mse_loss(op_info, device, dtype, requires_grad, **kwargs):
         for shape, kwargs in shapes_and_kwargs
     ]
 
+def sample_inputs_l1_loss(op_info, device, dtype, requires_grad, **kwargs):
+    sample_inputs = sample_inputs_loss(op_info, device, dtype, requires_grad, **kwargs)
+
+    if not dtype.is_complex:
+        return sample_inputs
+
+    make = partial(make_tensor, (), device=device, requires_grad=requires_grad)
+    float_dtype = torch.float64
+
+    sample_inputs.extend(
+        [
+            SampleInput(make(dtype=dtype), args=(make(dtype=float_dtype),)),
+            SampleInput(make(dtype=float_dtype), args=(make(dtype=dtype),)),
+        ]
+    )
+
+    return sample_inputs
+
+def sample_inputs_smooth_l1_loss(op_info, device, dtype, requires_grad, **kwargs):
```

Comment:
Should we add comments that we are explicitly not testing deprecated args to avoid reader confusion in the future?